ivy/functional/frontends/tensorflow/tensor.py

```diff
@@ -22,3 +22,6 @@ def add(self, y, name="add"):
 
     def get_shape(self):
         return tf_frontend.Shape(input=self.data)
+
+    def sub(self, y, name="sub"):
```

Comment:
Here

Reply:
[Instance Methods](https://www.tensorflow.org/api_docs/python/tf/Tensor) reference is given  in issue #3616. Here it is mentioned  as `sub`. Can you please reconfirm this whenever possible, I just want to confirm before making these changes.
Thanks!